easybuild/easyconfigs/e/ESMF/ESMF-7.0.2-intel-2017b.eb

```diff
@@ -0,0 +1,31 @@
+name = 'ESMF'
+version = '7.0.2'
+
+homepage = 'http://sourceforge.net/projects/esmf'
+description = """The Earth System Modeling Framework (ESMF) is software for building and coupling weather,
+ climate, and related models."""
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+toolchainopts = {'usempi': True}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['%%(namelower)s_%s_src.tar.gz' % '_'.join(version.split('.'))]
+patches = ['ESMF-6.1.1_libopts.patch']
+checksums = [
+    '73824e007db6af518b1b8f7fdb8d6a106f999de22916d181455ebd7dec750c08',  # esmf_7_0_2_src.tar.gz
+    '3851627f07c32a7da55d99072d619942bd3a1d9dd002e1557716158e7aacdaf4',  # ESMF-6.1.1_libopts.patch
+]
+
+dependencies = [
+    ('netCDF', '4.4.1.1', '-HDF5-1.8.19'),
+    ('netCDF-Fortran', '4.4.4', '-HDF5-1.8.19'),
+    ('netCDF-C++4', '4.3.0', '-HDF5-1.8.19'),
```

Comment:
I really don't like this kind of suffixes. Is the new version really needed?

Reply:
The suffixes are required because ESMF required HDF5 1.8.x, yet we like to build on top of the newest netCDF* versions...